server/lib/device/humidity-sensor/humidity-sensor.command.js

```diff
@@ -0,0 +1,51 @@
+const logger = require('../../../utils/logger');
+const { NotFoundError, NoValuesFoundError } = require('../../../utils/coreErrors');
+const { DEVICE_FEATURE_UNITS } = require('../../../utils/constants');
+
+/**
+ * @description Get the average humidity in a room.
+ * @param {Object} message - The message sent by the user.
+ * @param {Object} classification - The classification calculated by the brain.
+ * @param {Object} context - The context object containing found variables in question.
+ * @example
+ * command(message, classification, context);
+ */
+async function command(message, classification, context) {
+  let humidityResult;
+  const roomEntity = classification.entities.find((entity) => entity.entity === 'room');
```

Comment:
A bit late, but this is not required here, you can move it to line 28, to avoid useless loop if not required.

Reply:
Yeah but linter doesn't love
https://eslint.org/docs/rules/no-case-declarations

Reply:
I'm idiot, fixed :)